.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -234,10 +238,10 @@ jobs:
             compiler_cache_path: /home/runner/.cache/sccache
             clang_format_path: /home/runner/.local/bin/clang-format
             cargo_dir: ~/.cargo
-          - name: Ubuntu 22.04 (gcc) Qt6
-            os: ubuntu-22.04
+          - name: Ubuntu 24.04 (gcc) Qt6
+            os: ubuntu-24.04
             qt_version: 6
-            aqt_version: '6.7.3'
+            aqt_version: '6.8.0'
```

Comment:
remaining issue appears to be with Qt 6.8 the linking order is incorrect with Qt6QuickControls2 being linked before our staticlib.

Weirdly didn't happen in Qt 6.7.x or for other platforms. And not really clear why it is linked before

Reply:
This should be fixed via #1109